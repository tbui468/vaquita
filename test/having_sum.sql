if exists drop database sol;
create database sol;
open sol;

create table planets (
                        name string, 
                        mass float, 
                        moons int,
                        ring_system bool
                    );

insert into planets (
                        name, 
                        mass, 
                        moons,
                        ring_system
                    ) 

values
        ("Venus", 2.32, 0, false),
        ("Earth", null, 1, true),
        ("Mars", 4.23, 2, true),
        ("Saturn", 4.23, 2, true);

select ring_system, sum(moons) from planets group by ring_system having sum(moons) > 3;
close sol;
drop database sol;
exit;
