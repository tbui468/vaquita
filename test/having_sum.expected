created database sol
opened database sol
created table planets
inserted 4 record(s) into planets
true, 5, 
closed database sol
dropped database sol
